langserver/handler.go

```diff
@@ -80,6 +80,7 @@ type Language struct {
 	FormatCommand      string            `yaml:"format-command" json:"formatCommand"`
 	FormatCanRange     bool              `yaml:"format-can-range" json:"formatCanRange"`
 	FormatStdin        bool              `yaml:"format-stdin" json:"formatStdin"`
+	FormatTrimFile     bool          `yaml:"format-trim-file" josn:"formatTrimFile"`
```

Comment:
There is a typo here
```suggestion
	FormatTrimFile     bool          `yaml:"format-trim-file" json:"formatTrimFile"`
```